lingua_franca/lang/common_data_pt.py

```diff
@@ -116,3 +148,174 @@
+    (1e12, "bilião"),
+    (1e18, 'trilião'),
+    (1e24, "quadrilião"),
+    (1e30, "quintilião"),
+    (1e36, "sextilião"),
+    (1e42, "septilião"),
+    (1e48, "octilião"),
+    (1e54, "nonilião"),
+    (1e60, "decilião"),
+    (1e66, "undecilião"),
+    (1e72, "duodecilião"),
+    (1e78, "tredecilião"),
+    (1e84, "quattuordecilião"),
+    (1e90, "quinquadecilião"),
+    (1e96, "sedecilião"),
+    (1e102, "septendecilião"),
+    (1e108, "octodecilião"),
+    (1e114, "novendecilião"),
+    (1e120, "vigintilião"),
+    (1e306, "unquinquagintilião"),
+    (1e312, "duoquinquagintilião"),
+    (1e336, "sesquinquagintilião"),
+    (1e366, "unsexagintilião")
+])
+
+_SHORT_SCALE_PT = OrderedDict([
+    (100, 'cem'),
+    (1000, 'mil'),
+    (1000000, 'milhão'),
+    (1e9, "bilião"),
+    (1e12, 'trilião'),
+    (1e15, "quadrilião"),
+    (1e18, "quintilião"),
+    (1e21, "sextilião"),
+    (1e24, "septilião"),
+    (1e27, "octilião"),
+    (1e30, "nonilião"),
+    (1e33, "decilião"),
+    (1e36, "undecilião"),
+    (1e39, "duodecilião"),
+    (1e42, "tredecilião"),
+    (1e45, "quattuordecilião"),
+    (1e48, "quinquadecilião"),
+    (1e51, "sedecilião"),
+    (1e54, "septendecilião"),
+    (1e57, "octodecilião"),
+    (1e60, "novendecilião"),
+    (1e63, "vigintilião"),
+    (1e66, "unvigintilião"),
+    (1e69, "uuovigintilião"),
+    (1e72, "tresvigintilião"),
+    (1e75, "quattuorvigintilião"),
+    (1e78, "quinquavigintilião"),
+    (1e81, "qesvigintilião"),
+    (1e84, "septemvigintilião"),
+    (1e87, "octovigintilião"),
+    (1e90, "novemvigintilião"),
+    (1e93, "trigintilião"),
+    (1e96, "untrigintilião"),
+    (1e99, "duotrigintilião"),
+    (1e102, "trestrigintilião"),
+    (1e105, "quattuortrigintilião"),
+    (1e108, "quinquatrigintilião"),
+    (1e111, "sestrigintilião"),
+    (1e114, "septentrigintilião"),
+    (1e117, "octotrigintilião"),
+    (1e120, "noventrigintilião"),
+    (1e123, "quadragintilião"),
+    (1e153, "quinquagintilião"),
+    (1e183, "sexagintilião"),
+    (1e213, "septuagintilião"),
+    (1e243, "octogintilião"),
+    (1e273, "nonagintilião"),
+    (1e303, "centilião"),
+    (1e306, "uncentilião"),
+    (1e309, "duocentilião"),
+    (1e312, "trescentilião"),
+    (1e333, "decicentilião"),
+    (1e336, "undecicentilião"),
+    (1e363, "viginticentilião"),
+    (1e366, "unviginticentilião"),
+    (1e393, "trigintacentilião"),
+    (1e423, "quadragintacentilião"),
+    (1e453, "quinquagintacentilião"),
+    (1e483, "sexagintacentilião"),
+    (1e513, "septuagintacentilião"),
+    (1e543, "ctogintacentilião"),
+    (1e573, "nonagintacentilião"),
+    (1e603, "ducentilião"),
+    (1e903, "trecentilião"),
+    (1e1203, "quadringentilião"),
+    (1e1503, "quingentilião"),
+    (1e1803, "sescentilião"),
+    (1e2103, "septingentilião"),
+    (1e2403, "octingentilião"),
+    (1e2703, "nongentilião"),
+    (1e3003, "millinilião")
+])
+
+_ORDINAL_BASE_PT = {
+    1: 'primeiro',
+    2: 'segundo',
+    3: 'terceiro',
```

Comment:
On `_PT_NUMBERS` there are also these ordinals. I guess those should be deleted and let them just here